task/buildah-remote/0.2/buildah-remote.yaml

```diff
@@ -420,48 +438,52 @@ spec:
       # Needed to generate base images SBOM
       echo "$BASE_IMAGES" > /shared/base_images_from_dockerfile
 
-      buildah push "$IMAGE" oci:rhtap-final-image
+      buildah push "$IMAGE" "oci:konflux-final-image:$IMAGE"
```

Comment:
why the switch to `oci:$path:$image`?

Reply:
One side effect of changing the step image from `quay.io/redhat-appstudio/multi-platform-runner` to `quay.io/konflux-ci/buildah-task` is that we have picked up a newer version of `buildah`

<details>

```
$ podman run --tls-verify=false --rm -it --platform=linux/arm64 quay.io/redhat-appstudio/multi-platform-runner:01c7670e81d5120347cf0ad13372742489985e5f  buildah version
WARNING: image platform (linux/amd64) does not match the expected platform (linux/arm64)
Version:         1.29.1
Go Version:      go1.19.6
Image Spec:      1.0.2-dev
Runtime Spec:    1.0.2-dev
CNI Spec:        1.0.0
libcni Version:  v1.1.2
image Version:   5.24.1
Git Commit:
Built:           Mon Apr  3 08:34:42 2023
OS/Arch:         linux/amd64
BuildPlatform:   linux/amd64
$ podman run --tls-verify=false --rm -it --platform=linux/arm64 quay.io/konflux-ci/buildah-task:latest buildah version
Version:         1.38.0-dev
Go Version:      go1.21.11 (Red Hat 1.21.11-1.el9_4)
Image Spec:      1.1.0
Runtime Spec:    1.2.0
CNI Spec:        1.1.0
libcni Version:  v1.2.3
image Version:   5.32.0
Git Commit:
Built:           Wed Aug  7 13:15:20 2024
OS/Arch:         linux/arm64
BuildPlatform:   linux/arm64/v8
```

</details>

This included a change https://github.com/containers/common/commit/08fc0b450 where file paths are not valid tags. This resulted in the image being pulled but no repository/tags getting pulled with it. To resolve the issue, we had to properly push and pull the oci artifact's tag from its file path.

I can move this change to the commit where I change the buildah image if it would make it clearer.

Reply:
> I can move this change to the commit where I change the buildah image if it would make it clearer.

that would be nice :+1:

Reply:
This is now in the second commit and I have added more information to the commit message as well.